modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectLayoutUtil.java

```diff
@@ -33,13 +35,18 @@ public class ObjectLayoutUtil {
 
 	public static ObjectLayout toObjectLayout(
 		Map<String, Map<String, String>> actions,
+		ObjectDefinitionLocalService objectDefinitionLocalService,
 		ObjectFieldLocalService objectFieldLocalService,
 		com.liferay.object.model.ObjectLayout serviceBuilderObjectLayout) {
 
 		if (serviceBuilderObjectLayout == null) {
 			return null;
 		}
 
+		ObjectDefinition objectDefinition =
+			objectDefinitionLocalService.fetchObjectDefinition(
```

Comment:
@seltong can you replace by `getObjectDefinition`? Thanks

We should always use "get" methods if the code is supposed to work with that returned variable. Because that way it will raise an exception if it doesn't exist. 

cc @p-albuquerque

Reply:
Makes sense, thanks for clearing that up